none of the consumers in our tree want a GUI message pump (gtk or X or anything
else), but rather just want a glib message pump.  so change the UI define to go
straight to glib.  this lets us avoid gtk/X completely in our packages.

--- a/message_loop.cc
+++ b/message_loop.cc
@@ -153,7 +153,7 @@ MessageLoop::MessageLoop(Type type)
 #define MESSAGE_PUMP_UI NULL
 #define MESSAGE_PUMP_IO NULL
 #elif defined(OS_POSIX)  // POSIX but not MACOSX.
-#define MESSAGE_PUMP_UI new base::MessagePumpForUI()
+#define MESSAGE_PUMP_UI new base::MessagePumpGlib()
 #define MESSAGE_PUMP_IO new base::MessagePumpLibevent()
 #else
 #error Not implemented
